tests/test_cli.py

```
@@ -373,7 +373,7 @@ def test_parse_command_with_options(mock_parse_command, cli_runner):
     # Create a temporary directory for testing
     with tempfile.TemporaryDirectory() as tmpdir:
         result = cli_runner.invoke(
-            cli, ["parse", tmpdir, "--dashboard-name", "Custom Dashboard", "--max-files", "10", "--max-lines", "200"]
+            cli, ["parse", tmpdir, "--job-name", "api_logs", "--max-files", "10", "--max-lines", "200"]
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)